test/keys.js

```diff
@@ -10,7 +10,16 @@ describe('keys', function() {
   var cobj = new C();
 
   it("returns an array of the given object's own keys", function() {
-    eq(R.keys(obj).sort(), ['a', 'b', 'c', 'd', 'e', 'f']);
+    eq(R.keys(obj), ['a', 'b', 'c', 'd', 'e', 'f']);
+  });
+
+
+  it('lists the keys in their natural sort order', function() {
+    eq(R.keys({a: 1, d: 2, c: 3, e: 4, b: 5}), ['a', 'b', 'c', 'd', 'e']);
+  });
+
+  it('places numeric indices first, but in lexicographic order', function() {
```

Comment:
This is misleading, since numeric property names are not given special treatment:

```javascript
> R.keys({'0': true, '/': true})
['/', '0']
```

I suggest we remove the reference to numeric indices.